### ISSUE TITLE
Pipeline_Build_Test_JDK11_x86-64_linux Pipeline-Build-Test-Any-Platform.groovy not found

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#6561]
https://ci.eclipse.org/openj9/job/Pipeline_Build_Test_JDK11_x86-64_linux/519/
OpenJ9: 8300d8b
```
Started by upstream project "Pipeline-Build-Test-All" build number 586
originally caused by:
 Started by timer
Checking out git https://github.com/eclipse/openj9.git into /home/hudson/genie.openj9/.jenkins/jobs/Pipeline_Build_Test_JDK11_x86-64_linux/workspace@script to read buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-Any-Platform.groovy
No credentials specified
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/eclipse/openj9.git # timeout=10
Fetching upstream changes from https://github.com/eclipse/openj9.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/eclipse/openj9.git 
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/refs/heads/master^{commit} # timeout=10
Checking out Revision b39fa065a98484fad880c97c9d51cc04df41129e (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f b39fa065a98484fad880c97c9d51cc04df41129e
Commit message: "Merge pull request #3350 from acrowthe/add_DDR_modularity_tests"
 > git rev-list --no-walk b39fa065a98484fad880c97c9d51cc04df41129e # timeout=10
ERROR: /home/hudson/genie.openj9/.jenkins/jobs/Pipeline_Build_Test_JDK11_x86-64_linux/workspace@script/buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-Any-Platform.groovy not found
Finished: FAILURE
```

Last working OpenJ9: aaab6f5

**Comment by pshipton** [comment]
Also failing in OMR acceptance builds since https://ci.eclipse.org/openj9/job/Pipeline-OMR-Acceptance/1522/
OpenJ9: e3ec8ba

**Comment by pshipton** [comment]
https://github.com/eclipse/openj9/compare/aaab6f5...e3ec8ba

**Comment by pshipton** [comment]
@jdekonin @AdamBrousseau

**Comment by jdekonin** [comment]
I believe this is the issue that #6343 resolves.  @vsebe can you confirm?

**Comment by pshipton** [comment]
Merged #6343, and testing in https://ci.eclipse.org/openj9/view/Pipelines/job/Pipeline-OMR-Acceptance/1525/

**Comment by pshipton** [comment]
The builds has progressed further so I'll tentatively close this.
